pkg/kstatus/observe/observer/aggregator.go

```diff
@@ -0,0 +1,20 @@
+package observer
+
+import (
+	"sigs.k8s.io/cli-utils/pkg/kstatus/observe/event"
+	"sigs.k8s.io/cli-utils/pkg/kstatus/status"
+)
+
+// StatusAggregator provides the interface the observer uses to compute the aggregate status.
```

Comment:
// StatusAggregator is implemented by (resource status reporters?) so the Observer can aggregate their status (across what?)